ios/FluentUI/Tooltip/TooltipViewController.swift

```diff
@@ -40,51 +40,45 @@ class TooltipView: UIView {
             titleLabel.textAlignment = textAlignment
         }
 
-        addSubview(textContainer)
+        view.addSubview(textContainer)
 
         // Shadow
-        layer.insertSublayer(CALayer(), at: 0)
-        layer.insertSublayer(CALayer(), at: 0)
+        view.layer.insertSublayer(CALayer(), at: 0)
+        view.layer.insertSublayer(CALayer(), at: 0)
         updateShadows(tokenSet: tokenSet)
     }
 
     required init?(coder aDecoder: NSCoder) {
         preconditionFailure("init(coder:) has not been implemented")
     }
 
-    override func layoutSubviews() {
-        super.layoutSubviews()
+    override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
+        super.viewWillTransition(to: size, with: coordinator)
 
-        backgroundView.frame = bounds
-
-        if positionController.arrowDirection.isVertical {
-            arrowImageView.frame.origin.x = positionController.arrowPosition
-            backgroundView.frame.size.height -= arrowImageView.frame.height
-        } else {
-            arrowImageView.frame.origin.y = positionController.arrowPosition
-            backgroundView.frame.size.width -= arrowImageView.frame.width
+        coordinator.animate(alongsideTransition: nil) { _ in
+            self.updateAppearance(tokenSet: self.tokenSet)
```

Comment:
how noisy is this call? anything we should optimize?

Reply:
does it need to call the whole update appearance or just part that frame needs to update?